File -> Open Document gives:
    (mmass:5121): Gtk-WARNING **: 12:32:48.388: Failed to measure available space: The specified location is not supported
In doc.py, line 550, we changed the strings into bytes strings
    This may affect loading of the zlib compressed objects again
